remote/src/main/java/com/playone/mobile/remote/mapper/Extension.kt

```diff
@@ -7,16 +9,16 @@ import com.playone.mobile.remote.model.NotificationPayloadModel
 import com.playone.mobile.remote.model.PlayoneModel
 import com.playone.mobile.remote.model.UserModel
 
-fun PlayoneEntity.toModel(mapper: PlayoneEntityMapper) = mapper.mapFromData(this)
+inline fun PlayoneEntity.toModel(mapper: PlayoneEntityMapper) = mapper.mapFromData(this)
```

Comment:
Since there is no lambda function in the parameter, what you think? use `internal fun` instead of inline function? As you can see the warning, `inline work best for functions with lambda parameters`

Reply:
I know `inline function` is better for lambda parameters. But I don't expect `inline` works for `lambda parameters` here.